frontend/src/components/common/form/FastFiscalYearInput.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { memo, useEffect } from 'react';
```

Comment:
masked input that saves/accepts YYYY format from the DB but displays in YYYY/YYYY format.